torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1868,6 +1875,13 @@ def _simplify_floor_div(self, expr):
             self.evaluate_expr(eq_expr)
         return self.simplify(expr)
 
+    def _add_guard(self, expr: "sympy.Expr") -> None:
+        stack = ''.join(traceback.format_list(traceback.extract_stack()[:-1]))
```

Comment:
this changed from `:-2` to `:-1`

Reply:
It is right to renumber but you want to go to -3 not -1

Reply:
I don't understand. -1 cuts only the last frame, which is the one calculating the stack; on the other hand it gives the stack up to the caller of `_add_guard` . This should be useful. It certainly is for the other place, because there are multiple different callers of `_set_replacement`. 

If anything I think we should snip from the front, because the first couple of entries are of the form `File "<string>" ...` and the next couple are from `runpy.py`.